### PR TITLE
[TRANSLATION] Improve Dutch (nl-NL) translation

### PR DESCRIPTION
**Pull request by GoombaProgrammer**
## Purpose

Improve the Dutch translation in Task Manager, Notepad, Explorer and Welcome.

### CODE REVIEW

File: base/applications/notepad/lang/nl-NL.rc [hunk 1]
```diff
@@ -24,7 +24,7 @@ BEGIN
     POPUP "&Bestand"
     BEGIN
         MENUITEM "&Nieuw\tCtrl+N", CMD_NEW
-        MENUITEM "New &Window\tCtrl+Shift+N", CMD_NEW_WINDOW
+        MENUITEM "Nieu&w Venster\tCtrl+Shift+N", CMD_NEW_WINDOW
```
Comment (HBelusca) [inline base/applications/notepad/lang/nl-NL.rc:27]:
```suggestion
        MENUITEM "Nieuw &Venster\tCtrl+Shift+N", CMD_NEW_WINDOW
```

**Review by HBelusca — COMMENTED** [review]
There is still some remaining english in the files you've touched.

File: base/applications/notepad/lang/nl-NL.rc [hunk 2]
```diff
@@ -24,7 +24,7 @@ BEGIN
     POPUP "&Bestand"
     BEGIN
         MENUITEM "&Nieuw\tCtrl+N", CMD_NEW
-        MENUITEM "New &Window\tCtrl+Shift+N", CMD_NEW_WINDOW
+        MENUITEM "Nieuw &Venster\tCtrl+Shift+N", CMD_NEW_WINDOW
```
Comment (learn-more) [inline base/applications/notepad/lang/nl-NL.rc:27]: venster hoeft niet met een hoofdletter

File: base/applications/notepad/lang/nl-NL.rc [hunk 3]
```diff
@@ -45,8 +45,8 @@ BEGIN
         MENUITEM SEPARATOR
         MENUITEM "&Zoeken\tCtrl+Z", CMD_SEARCH
         MENUITEM "&Volgende zoeken\tF3", CMD_SEARCH_NEXT
-        MENUITEM "Replace...\tCtrl+H", CMD_REPLACE
-        MENUITEM "Go To...\tCtrl+G", CMD_GOTO
+        MENUITEM "Vervangen...\tCtrl+H", CMD_REPLACE
+        MENUITEM "Ga Naar...\tCtrl+G", CMD_GOTO
```
Comment (learn-more) [inline base/applications/notepad/lang/nl-NL.rc:49]: naar hoeft niet met een hoofletter

File: base/applications/notepad/lang/nl-NL.rc [hunk 4]
```diff
@@ -121,12 +121,12 @@ END
 DIALOG_GOTO DIALOGEX 0, 0, 165, 50
 STYLE DS_SHELLFONT | DS_MODALFRAME | WS_CAPTION | WS_SYSMENU
 FONT 8, "MS Shell Dlg"
-CAPTION "Goto line"
+CAPTION "Ga naar lijn"
```
Comment (learn-more) [inline base/applications/notepad/lang/nl-NL.rc:124]: 'regel' is een normalere vertaling

File: base/applications/taskmgr/lang/nl-NL.rc [hunk 5]
```diff
@@ -46,17 +46,17 @@ BEGIN
...
-        MENUITEM "&Log Off %s",                ID_SHUTDOWN_LOGOFF
-        MENUITEM "&Switch User\tWinKey+L",     ID_SHUTDOWN_SWITCHUSER, GRAYED
-        MENUITEM "Loc&k Computer\tWinKey+L",   ID_SHUTDOWN_LOCKCOMPUTER
-        MENUITEM "&Disconnect",                ID_SHUTDOWN_DISCONNECT, GRAYED
-        MENUITEM "&Eject Computer",            ID_SHUTDOWN_EJECT_COMPUTER, GRAYED
+        MENUITEM "Stand &By",                           ID_SHUTDOWN_STANDBY
+        MENUITEM "&Hibernate",                          ID_SHUTDOWN_HIBERNATE
+        MENUITEM "&Uitzetten",                          ID_SHUTDOWN_POWEROFF
+        MENUITEM "&Opnieuw Opstarten",                  ID_SHUTDOWN_REBOOT
+        MENUITEM "&%s Uitloggen",                       ID_SHUTDOWN_LOGOFF
```
Comment (learn-more) [inline base/applications/taskmgr/lang/nl-NL.rc:55]: Ik denk dat 'afmelden' vaker gebruikt wordt?

File: base/shell/explorer/lang/nl-NL.rc [hunk 6]
```diff
@@ -5,14 +5,14 @@ BEGIN
     POPUP ""
     BEGIN
         MENUITEM SEPARATOR
-        MENUITEM "&Adjust Date/Time", ID_SHELL_CMD_ADJUST_DAT
-        MENUITEM "&Customize Notifications...", ID_SHELL_CMD_CUST_NOTIF
+        MENUITEM "Ver&ander Datum/Tijd", ID_SHELL_CMD_ADJUST_DAT
+        MENUITEM "&Notificaties Aanpassen...", ID_SHELL_CMD_CUST_NOTIF
```
Comment (learn-more) [inline base/shell/explorer/lang/nl-NL.rc:9]: `Meldingen` ipv `Notificaties`

**Review by learn-more — DISMISSED** [review]
Hello,
Thanks for your interest in ReactOS!
We ask our contributors [to use their full name when committing](https://github.com/reactos/reactos/blob/master/CONTRIBUTING.md#rules-and-recommendations).
[Please amend your commit with your full name](https://stackoverflow.com/a/3042512), and update this PR.

To change this for future PR's, you can update it for globally (for the system):
```
git config --global user.name "Your Name"
git config --global user.email you@example.com
```

Or just for the current repository:
```
git config user.name "Your Name"
git config user.email you@example.com
```

Regards,

**Comment by binarymaster** [comment]
@GoombaProgrammer it seems you are inactive, would it be okay if I take over and finish this PR?

I already got your full name and email information from your other commit in different repository on GitHub, which I would then apply here, because it's required by our contribution rules.

Please answer here if you're okay with that. Thanks in advance.

**Comment by GoombaProgrammer** [comment]
@binarymaster Yeah, you can

File: base/applications/taskmgr/lang/nl-NL.rc [hunk 7]
```diff
@@ -46,17 +46,17 @@ BEGIN
...
+        MENUITEM "&Hibernate",                          ID_SHUTDOWN_HIBERNATE
+        MENUITEM "&Uitzetten",                          ID_SHUTDOWN_POWEROFF
+        MENUITEM "&Opnieuw Opstarten",                  ID_SHUTDOWN_REBOOT
+        MENUITEM "%s &afmelden",                        ID_SHUTDOWN_LOGOFF
+        MENUITEM "&Verander van gebruiker\tWinKey+L",   ID_SHUTDOWN_SWITCHUSER, GRAYED
+        MENUITEM "Loc&k Computer\tWinKey+L",            ID_SHUTDOWN_LOCKCOMPUTER
+        MENUITEM "&Disconnect",                         ID_SHUTDOWN_DISCONNECT, GRAYED
+        MENUITEM "&Eject Computer",                     ID_SHUTDOWN_EJECT_COMPUTER, GRAYED
     END
     POPUP "&Help"
```
Comment (HBelusca) [inline base/applications/taskmgr/lang/nl-NL.rc:61]: Those last three lines haven't been translated, as well as the "Help" popup.

File: base/shell/explorer/lang/nl-NL.rc [hunk 8]
```diff
@@ -5,14 +5,14 @@ BEGIN
...
     BEGIN
         MENUITEM SEPARATOR
-        MENUITEM "&Adjust Date/Time", ID_SHELL_CMD_ADJUST_DAT
-        MENUITEM "&Customize Notifications...", ID_SHELL_CMD_CUST_NOTIF
+        MENUITEM "Ver&ander Datum/Tijd", ID_SHELL_CMD_ADJUST_DAT
+        MENUITEM "&Meldingen aanpassen...", ID_SHELL_CMD_CUST_NOTIF
         MENUITEM SEPARATOR
         MENUITEM "Ca&scade Windows", ID_SHELL_CMD_CASCADE_WND
         MENUITEM "Tile Windows &Horizontally", ID_SHELL_CMD_TILE_WND_H
         MENUITEM "Tile Windows V&ertically", ID_SHELL_CMD_TILE_WND_V
```
Comment (HBelusca) [inline base/shell/explorer/lang/nl-NL.rc:13]: Those three entries need to be translated as well.

File: base/applications/taskmgr/lang/nl-NL.rc (hunk 7 above)
Comment (binarymaster) [reply base/applications/taskmgr/lang/nl-NL.rc:61]: I don't think the PR author is active, as I already took over this PR -- but also I'm not a native speaker of this language. Maybe we can merge this as is, so everything else could be translated by someone else. But I'm open to suggestions, if somebody wants to help with translating these.

**Pull request merged**